src/modules/mim/daemonconfiguration.json

```diff
@@ -0,0 +1,143 @@
+{
+    "name": "DaemonConfigurationModel",
+    "type": "mimModel",
+    "contents": [
+      {	
+        "name": "DaemonConfiguration",
+        "type": "mimComponent",
+        "contents": [
+          {
+            "name": "daemons",
+            "type": "mimObject",
+            "desired": false,
+            "schema": {
+              "type": "array",
+              "elementSchema": {
+                "type": "object",
+                "fields": [
+                  {
+                    "name": "name",
+                    "schema": "string"
+                  },
+                  {
+                    "name": "state",
+                    "schema": {
+                      "type": "enum",
+                      "valueSchema": "integer",
+                      "enumValues": [
+                        {
+                          "name": "unknown",
+                          "enumValue": 0
+                        },
+                        {
+                          "name": "running",
+                          "enumValue": 1
+                        },
+                        {
+                          "name": "failed",
+                          "enumValue": 2
+                        },
+                        {
+                          "name": "exited",
+                          "enumValue": 3
+                        },
+                        {
+                          "name": "dead",
+                          "enumValue": 4
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "name": "status",
```

Comment:
"state" and "status" here for reported object may need to be separate indeed but the names are basically the same. Either think of more different names or merge the two into a single enum, like suggested for the desired object "desired state".